app/views/markets/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<nav>
+<a class="nav_link" href="/">Return Home | </a>
```

Comment:
Watch your whitespace in this file. There are a lot of nested tags that do not have any nested indentation